Library/Formula/mapserver.rb

```diff
@@ -31,6 +30,8 @@ class Mapserver < Formula
   depends_on 'fcgi' if build.with? "fastcgi"
   depends_on 'cairo' => :optional
 
+  patch :DATA
```

Comment:
Can you add a comment explaining what this patch does and where it's been reported or committed upstream? Helps us know when it's safe to get rid of. Thanks!